indico/modules/events/editing/templates/management/editable_type.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'events/editing/management/base.html' %}
+
+{% block subtitle %}
+    {% trans t=type|capitalize %} {{ t }} {% endtrans %}
```

Comment:
The `trans` here doesn't do anything useful. Change the `EditableType` enum to a `RichIntEnum` (many examples in the codebase) where you can set titles for the elements (which can be marked for translation). Then pass the `.title` of the enum member (or rather the enum member itself since you need both the `.title` and the `.name`) to this template, and just use the title here directly - no need for wrapping it in trans!